src/game/handlers/charInfo.js

```diff
@@ -13,6 +25,29 @@ import { getCurrentStatPoints } from './statHelpers'
 import { rejectUndefined } from './errors'
 import { capitalize } from './helpers'
 
+import models from '../models'
+
+function getEquippedIds (character) {
+  return filter(pipe(isNil, not), values(character.equips))
+}
+
+function getEquipDetails (equipIds) {
```

Comment:
I added the "get" because I usually like having a verb starting my functions.

You guys should try to do a basic style guidelines for your project, it would help people making more quality pull requests.